specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/GalleryRP.json

```diff
@@ -5485,6 +6242,273 @@
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "GalleryScriptVersionList": {
+      "type": "object",
+      "description": "The List Gallery Script version operation response.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The GalleryScriptVersion items on this page",
+          "items": {
+            "$ref": "#/definitions/GalleryScriptVersion"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "GalleryScriptVersionProperties": {
+      "type": "object",
+      "description": "Describes the properties of a gallery script version.",
+      "properties": {
+        "publishingProfile": {
+          "$ref": "#/definitions/GalleryScriptVersionPublishingProfile",
+          "description": "The publishing profile of a gallery image version."
+        },
+        "safetyProfile": {
+          "$ref": "#/definitions/GalleryScriptVersionSafetyProfile",
+          "description": "The safety profile of the Gallery Script Version."
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/GalleryProvisioningState",
+          "description": "The provisioning state, which only appears in the response.",
+          "readOnly": true
+        },
+        "replicationStatus": {
+          "$ref": "#/definitions/ReplicationStatus",
+          "description": "This is the replication status of the gallery image version.",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "publishingProfile"
+      ]
+    },
+    "GalleryScriptVersionPublishingProfile": {
+      "type": "object",
+      "description": "The publishing profile of a gallery image version.",
+      "properties": {
+        "source": {
+          "$ref": "#/definitions/ScriptSource",
+          "description": "The source script from which the Script Version is going to be created."
+        }
+      },
+      "required": [
+        "source"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/GalleryArtifactPublishingProfileBase"
+        }
+      ]
+    },
+    "GalleryScriptVersionSafetyProfile": {
+      "type": "object",
+      "description": "The safety profile of the Gallery Script Version.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/GalleryArtifactSafetyProfileBase"
+        }
+      ]
+    },
+    "GalleryScriptVersionUpdate": {
+      "type": "object",
+      "description": "Specifies information about the gallery Script Version that you want to update.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GalleryScriptVersionProperties",
```

Comment:
same issue as GalleryScriptUpdate. 
Should we create a model called GalleryScriptVersionUpdateProperties?